src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
+
+set BridgeHost=
+cd %2
```

Comment:
Consider `pushd` and `popd` - someone running the script should not have their current working directory change after script runs.